overflow/github.md

```diff
@@ -0,0 +1,95 @@
+Title | Tags
+--- | ---
+Intro to GitHub | github, git
+
+## Objectives
+* Learn GitHub basics
+* Launch a GitHub pages project
+
+## TODO: Structure
+* *This is just an example structure, feel free to change*
+* 10:00 - 10:40 - Challenge problem
+* 10:40 - 10:45 Break
+* 10:45 - 11:30 Lecture part 1
+* 11:30 - 1:00 Exercise 1
+* 1:00 - 2:00 Lunch
+* 2:00 - 2:55 Lecture part 2
+* 2:55 - 3:00 Break
+* 3:00 - 4:55 Exercise 2
+* 4:55 - 5:00  Break
+* 5:00 - 6:00 - Review 
```

Comment:
SHould we take the structure from Rina's docs on time breakdown? And not have a custom structure for each class?